OpenUtau/ViewModels/UpdaterViewModel.cs

```diff
@@ -92,7 +92,7 @@ public UpdaterViewModel() {
                 return null;
             }
             return releases
-                .Where(r => !r.draft && r.prerelease == Preferences.Default.Beta)
+                .Where(r => !r.draft && (!r.prerelease || r.prerelease == Preferences.Default.Beta))
```

Comment:
Could simplify logic:
```suggestion
                .Where(r => !r.draft && (!r.prerelease || Preferences.Default.Beta))
```